src/daemon/bundle.rs

```diff
@@ -1,33 +1,45 @@
 // Copyright 2019-2023 ChainSafe Systems
 // SPDX-License-Identifier: Apache-2.0, MIT
 
-use crate::utils::db::{car_stream::CarHeader, car_util::load_car};
-use anyhow::Context as _;
+use crate::{
+    networks::{ActorBundleInfo, ACTOR_BUNDLES},
+    utils::{db::car_util::load_car, net::http_get},
+};
+use anyhow::ensure;
+use futures::{stream::FuturesUnordered, TryStreamExt};
 use fvm_ipld_blockstore::Blockstore;
+use std::io::Cursor;
+use tracing::warn;
 
-pub async fn load_actor_bundles(db: &impl Blockstore) -> anyhow::Result<CarHeader> {
-    const ERROR_MESSAGE: &str = "Actor bundles assets are not properly downloaded, make sure git-lfs is installed and run `git lfs pull` again. See <https://github.com/git-lfs/git-lfs/blob/main/INSTALLING.md>";
+/// Tries to load the missing actor bundles to the blockstore. If the bundle is
+/// not present, it will be downloaded.
+pub async fn load_actor_bundles(db: &impl Blockstore) -> anyhow::Result<()> {
+    FuturesUnordered::from_iter(
+        ACTOR_BUNDLES
+            .iter()
+            .filter(|bundle| !db.has(&bundle.manifest).unwrap_or(false))
+            .map(
+                |ActorBundleInfo {
+                     manifest: root,
+                     url,
+                     alt_url,
+                 }| async move {
+                    let response = if let Ok(response) = http_get(url).await {
+                        response
+                    } else {
+                        warn!("failed to download bundle from primary URL, trying alternative URL");
+                        http_get(alt_url).await?
+                    };
```

Comment:
I'm kinda worried that we'll forget to mirror the files. How about we download the files from both sources and print a warning if one mirror is down (and an error if the files are different)?

Reply:
https://github.com/ChainSafe/forest/pull/3622/commits/2b27412e2e10c4edd15387962b7a280d1962011b

@lemmih Does it look okay for you?